### PR TITLE
Update Linux instructions

### PR DESCRIPTION
**Pull request by UK992**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -83,7 +83,7 @@ <h2 class='header'>Linux Instructions</h2>
                     <li>Download the .TAR.GZ file</li>
                     <li>Expand the file (`tar zxf servo-latest.tar.gz`)</li>
                     <li>`cd servo`</li>
-                    <li>Execute the `./runservo.sh` script to run Servo</li>
+                    <li>Execute the `./servo` script to run Servo</li>
```
Comment (aneeshusa) [inline index.html]: `./servo` isn't a script but the actual executable, so remove `the` and `script` from this line.

**Review by aneeshusa — CHANGES_REQUESTED** [review]
One nit.

**Comment by UK992** [comment]
done

**Review by aneeshusa — APPROVED** [review]
Thanks for the PR, LGTM! Homu is [not yet enabled](https://github.com/servo/saltfs/pull/574) for this repo, so I can't merge this yet.

**Comment by jdm** [comment]
cc @edunham 
We're missing homu and/or other developers in the teams allowed to merge things in this repository.

**Comment by aneeshusa** [comment]
@jdm that work is already underway, see https://github.com/servo/saltfs/pull/574 and #6, this repo just needs a .travis.yml file.

**Pull request merged**